openmc/mgxs/library.py

```diff
@@ -107,6 +111,8 @@ def __init__(self, openmc_geometry, by_nuclide=False,
         self._domain_type = None
         self._domains = 'all'
         self._energy_groups = None
+        self._num_polar = None
```

Comment:
As for the re-organization, dont forget delayed groups arent an energy quantity.